packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -142,58 +219,96 @@ function VisXZoom({
       scaleXMax={zoomConfiguration.maxZoom}
       scaleYMin={zoomConfiguration.minZoom}
       scaleYMax={zoomConfiguration.maxZoom}
-      passive
       top={top}
       width={width}
+      wheelDelta={wheelDelta}
     >
       {_zoom => {
         zoomRef.current = _zoom
         return (
-          <ZoomingComponent
-            initialTransformMatrix={_zoom.initialTransformMatrix}
-            transformMatrix={_zoom.transformMatrix}
-            transform={_zoom.toString()}
-            {...props}
-          >
-            <ZoomEventLayer
-              focusable
-              height={height}
-              onDoubleClick={onDoubleClick}
-              onKeyDown={onKeyZoom}
-              onPointerEnter={onPointerEnter}
-              onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
-              onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
-              onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
-              onPointerLeave={onPointerLeave}
-              onWheel={throttledOnWheel}
-              panning={panning}
-              tabIndex={0}
-              width={width}
-            />
-          </ZoomingComponent>
+          <g ref={wheelEventLayer}>
+            <ZoomingComponent
+              initialTransformMatrix={_zoom.initialTransformMatrix}
+              move={move}
```

Comment:
I accidentally left this in from an earlier version of this PR. It doesn't do anything. 

```suggestion
```

Reply:
Fixed!